### PR TITLE
:bug: fix issue with select menu getting cut by edge of modal

### PR DESCRIPTION
**Pull request by gitdallas**
image

becomes

image

### CODE REVIEW

File: client/src/app/components/SimpleSelect.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export const SimpleSelect: React.FC<ISimpleSelectProps> = ({
 
   return (
     <Select
+      menuAppendTo="parent" // prevent menu from being clipped by modal edges
```
Comment (ibolton336) [inline client/src/app/components/SimpleSelect.tsx:43]: Adding this overflow seems to be a much better experience, but not sure if it is the PF friendly way to accomplish this. 
```suggestion
      menuAppendTo="parent" // prevent menu from being clipped by modal edges
      style={{ overflowY: "auto", maxHeight: "200px" }}

```

**Review by ibolton336 — CHANGES_REQUESTED** [review]

File: client/src/app/components/SimpleSelect.tsx (hunk 1 above)
Comment (ibolton336) [reply client/src/app/components/SimpleSelect.tsx:43]: Screenshot 2023-10-03 at 7 54 02 PM

Comment (gitdallas) [reply client/src/app/components/SimpleSelect.tsx:43]: i added a maxHeight of 200

**Comment by gitdallas** [comment]
@ibolton336 overflow is automatic

File: client/src/app/components/SimpleSelect.tsx (hunk 1 above)
Comment (ibolton336) [reply client/src/app/components/SimpleSelect.tsx:43]: I see the overflow now with the maxHeight set. LGTM other than the console.log

**Review by ibolton336 — CHANGES_REQUESTED** [review]

File: client/src/app/components/SimpleSelect.tsx [hunk 2]
```diff
@@ -37,9 +37,11 @@ export const SimpleSelect: React.FC<ISimpleSelectProps> = ({
   ...props
 }) => {
   const [isOpen, setIsOpen] = useState(false);
-
+  console.log({ ...props });
```
Comment (ibolton336) [inline client/src/app/components/SimpleSelect.tsx:40]: This guy snuck in. Otherwise LGTM.

**Review by ibolton336 — APPROVED** [review]

**Pull request merged**